src/pages/AddItem.jsx

```diff
@@ -1,3 +1,245 @@
+                  className="h-4 w-4"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  stroke="currentColor"
+                  strokeWidth={2}
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M6 18L18 6M6 6l12 12"
+                  />
+                </svg>
+              </button>
+            </div>
+          )}
+        </div>
+        {(showWarning || (selectedImage && !showWarning)) && (
+          <p className="text-orange-600 text-sm mt-4">
+            * 이미지 등록은 최대 1개까지 가능합니다.
+          </p>
+        )}
+
+        <div>
+          <label
+            htmlFor="productName"
+            className="block text-[1.4rem] font-[700] text-gray-700 mb-4"
+          >
+            상품명
+          </label>
+          <Input
+            id="productName"
+            placeholder="상품명을 입력해주세요"
+            value={productName}
+            onChange={(e) => setProductName(e.target.value)}
+          />
+        </div>
+        <div>
+          <label
+            htmlFor="productDescription"
+            className="block text-[1.4rem] font-[700] text-gray-700 mb-1"
+          >
+            상품 소개
+          </label>
+          <textarea
+            id="productDescription"
+            placeholder="상품 소개를 입력해주세요"
+            rows="10"
+            className="w-full p-4 bg-[#F3F4F6] placeholder:text-[#9CA3AF]"
+            value={productDescription}
+            onChange={(e) => setProductDescription(e.target.value)}
+          />
+        </div>
+        <div>
+          <label
+            htmlFor="productPrice"
+            className="block text-[1.4rem] font-[700] text-gray-700 mb-1"
+          >
+            판매가격
+          </label>
+          <Input
+            id="productPrice"
+            type="number"
+            placeholder="판매 가격을 입력해주세요"
+            value={productPrice}
+            onChange={(e) => setProductPrice(e.target.value)}
+          />
+        </div>
+        <div>
+          <label
+            htmlFor="productTags"
+            className="block text-[1.4rem] font-[700] text-gray-700 mb-1"
+          >
+            태그
+          </label>
+          <Input
+            id="productTags"
+            placeholder="태그를 입력해주세요"
+            value={tagInput}
+            onChange={(e) => setTagInput(e.target.value)}
+            onKeyDown={handleTagKeyDown}
+          />
+          <div className="flex gap-2 flex-wrap mt-2">
+            {tags.map((tag) => (
+              <div
+                key={tag}
+                className="bg-gray-100 text-black px-3 py-2 rounded-full flex items-center text-[1.2rem]"
+              >
+                #{tag}
+                <div
+                  type="button"
+                  onClick={() => removeTag(tag)}
+                  className="ml-2 bg-[#9CA3AF] p-1 rounded-full"
+                >

```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 버튼을 사용해주세요. 더 적절한 태그가 있는데 div 태그에 onClick 을 주게되면 button에게 기대하는 동작을 제대로 수행하지 못합니다!
```suggestion
                <button
                  type="button"
                  onClick={() => removeTag(tag)}
                  className="ml-2 bg-[#9CA3AF] p-1 rounded-full"
                >

```